### PR TITLE
adjust L1 scc contract to accept signature and verify

### PR DESCRIPTION
**Pull request by silmaj**

### CODE REVIEW

File: bitnetwork.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chengwenxi) [inline bitnetwork.iml:1]: delete it?

File: packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol [hunk 2]
```diff
@@ -294,6 +320,35 @@ contract StateCommitmentChain is IStateCommitmentChain, Lib_AddressResolver {
...
+     */
+    function _distributeTssReward(bytes32[] calldata _batch, uint256 _shouldStartAtElement) internal {
+        // get address of tss group member
+        address[] memory tssMembers = ITSSGroupContract(resolve("TSSGroupContract")).GetTssMembers();
+        require(tssMembers.length > 0, "get tss members in error");
+
+        // construct calldata for claimReward call
+        bytes memory message = abi.encodeWithSelector(
+            ITssRewardContract.claimReward.selector,
+            _shouldStartAtElement,
```
Comment (FletcherMan) [inline packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol:336]: the `_shouldStartAtElement` may be not the same as the the `_blockStartHeight` in L2 TssRewardContract

File: bitnetwork.iml (hunk 1 above)
Comment (silmaj) [reply bitnetwork.iml:1]: done

File: packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol (hunk 2 above)
Comment (silmaj) [reply packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol:336]: if _shouldStartAtElement == L2 height, it will work.

**Pull request closed without merging**